src/systems/physics/Physics.cc

```diff
@@ -1028,6 +1043,54 @@ void PhysicsPrivate::CreatePhysicsEntities(const EntityComponentManager &_ecm)
               math::eigen3::convert(_pose->Data()),
               math::eigen3::convert(meshSdf->Scale()));
         }
+        else if (_geom->Data().Type() == sdf::GeometryType::HEIGHTMAP)
+        {
+          auto linkHeightmapFeature =
+              this->entityLinkMap.EntityCast<HeightmapFeatureList>(
+                  _parent->Data());
+          if (!linkHeightmapFeature)
+          {
+            static bool informed{false};
+            if (!informed)
+            {
+              igndbg << "Attempting to process heightmap geometries, but the physics"
```

Comment:
some code check complaints:

~~~
+ make codecheck
5606
  Scanning dependencies of target cpplint
5607
  /github/workspace/src/systems/physics/Physics.cc:1056:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
5608
  /github/workspace/src/systems/physics/Physics.cc:1058:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
5609
  /github/workspace/src/systems/physics/Physics.cc:1073:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
5610
  /github/workspace/src/systems/physics/Physics.cc:1088:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
5611
  /github/workspace/src/systems/physics/Physics.cc:1326:  Small and focused functions are preferred: PhysicsPrivate::CreatePhysicsEntities() has 527 non-comment lines (error triggered by exceeding 500 lines).  [readability/fn_size] [1]
5612
  Total errors found: 5
~~~

Reply:
9b0d908fa7a460ab0f72480d053e5165e7687358